doc/sphinx-guides/source/qa/testing-approach.md

```diff
@@ -34,7 +34,7 @@ Think about risk. Is the feature or function part of a critical area such as per
 
 ## Smoke Test
 
-1. Go to the homepage on <https://dataverse-internal.iq.harvard.edu>. Scroll to the bottom to ensure the build number is the one you intend to test from Jenkins.
+1. Go to the homepage on <https://qa.dataverse.org> (this server has production data). Scroll to the bottom to ensure the build number is the one you intend to test from Jenkins.
```

Comment:
Just a general comment that we could add more to the QA Guide, in general. I only changed this page because we link to it from "Conduct Regression Testing".